logstash-core/lib/logstash/api/lib/app/commands/stats/events_command.rb

```diff
@@ -8,9 +8,9 @@ def run
     #need to be tailored to the right metrics for this command.
     stats = service.get(:events_stats)
     {
-      :in => stats[:base][:events_in].value,
-      :out => 0,
-      :dropped => stats[:base][:events_filtered].value
+      :in => stats[:stats][:events][:in].value,
+      :out => stats[:stats][:events][:out].value,
+      :dropped => stats[:stats][:events][:filtered].value
```

Comment:
`Filtered` is a different beast, since an event IN the fitlers queue doesn't necessary mean and event out. I have added the filtered counter.

Also this counter will also be updated by the flush ticks.